src/scripts/main.js

```diff
@@ -1 +1,61 @@
+
 'use strict';
+
+function firstPromise() {
+  return new Promise((resolve, reject) => {
+    document.addEventListener('click', () => {
+      resolve('First promise was resolved');
+    });
+
+    setTimeout(() => {
+      reject(new Error('First promise was rejected'));
+    }, 3000);
+  });
+}
+
+function secondPromise() {
+  return new Promise(resolve => {
+    document.addEventListener('mousedown', (e) => {
```

Comment:
This mousedown event can be triggered not only by left/right button on the mouse. You can add some condition to detect propper click. Check https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons